### PR TITLE
Add Homebrew installation instructions to documentation.

### PR DESCRIPTION
**Pull request by chewrocca**
Hoping to close out https://github.com/grafana/grizzly/issues/228

### CODE REVIEW

File: docs/content/installation.md [hunk 1]
```diff
@@ -20,4 +20,15 @@ git clone https://github.com/grafana/grizzly.git
 cd grizzly
 make dev
 sudo mv grr /usr/local/bin/grr
-```
\ No newline at end of file
+```
+
+Installing Grizzly on macOS via Homebrew
```
Comment (malcolmholmes) [inline docs/content/installation.md:25]: Could you make this a heading? H2 I figure. And add an "Installing Grizzly on Linux" heading further up?

Comment (chewrocca) [reply docs/content/installation.md]: Added headers, added language for fenced blocks, and reordered document.

**Review by malcolmholmes — APPROVED** [review]

**Comment by tshuma1** [comment]
any progress on merging this PR?

**Comment by spinillos** [comment]
Thanks!

**Pull request merged**